packages/react-composites/src/composites/CallWithChatComposite/EmbeddedPeoplePane.tsx

```diff
@@ -131,7 +131,12 @@ export const EmbeddedPeoplePane = (props: {
     >
       <Stack tokens={peoplePaneContainerTokens}>
         {inviteLink && (
-          <DefaultButton text="Copy invite link" iconProps={{ iconName: 'Link' }} onClick={() => copy(inviteLink)} />
+          <DefaultButton
+            text="Copy invite link"
```

Comment:
should there be a localized string used here instead?

Reply:
Resolved by adding localized string:
text={callWithChatStrings.copyInviteLinkButtonLabel}